book/website/ethical-research/activism/activism-case-study-google.md

```diff
@@ -0,0 +1,53 @@
+(er-activism-case-study-google)=
+# Case study: Google Workers & Project Maven
+
+Tech workers have historically had low levels of unionisation for a variety of different reasons. 
+Some have also been ambivalent about the potential impact of the companies and products they work for on systemic inequality and injustice across the globe.
```

Comment:
Not sure if this is worded as reason for low levels of unionisation, or a separate issue?

Reply:
Good point @ninadicara I've changed this into a separate paragraph so it's a bit clearer